src/ImageSequencer.js

```diff
@@ -77,18 +90,23 @@ ImageSequencer = function ImageSequencer(options) {
     return this;
   }
 
+  /**
+   *
+   * @param {Object} ref ImageSequencer Reference
+   * @param {Number} index
+   */
   function removeStep(ref, index) {
-    //remove the step from images[image].steps and redraw remaining images
+    // remove the step from images[image].steps and redraw remaining images
```

Comment:
Be consistent in style. 
Somewhere you wrote inline comments while somewhere you wrote on new lines. Somewhere you wrote multi line comments.

Reply:
Ok. So what I'll do is:
- Simple utility methods -> inline
- Class methods or other important methods -> multiline JSDoc or normal comments. Is that ok?